packages/story-editor/src/components/floatingMenu/elements/shared/useVideoTranscoding.js

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { useLocalMedia } from '../../../../app';
+import useVideoTrim from '../../../videoTrim/useVideoTrim';
+import useFFmpeg from '../../../../app/media/utils/useFFmpeg';
+import useProperties from './useProperties';
+
+function useVideoTranscoding() {
```

Comment:
This hook could be very useful for [VideoOptionsPanel](https://github.com/GoogleForCreators/web-stories-wp/blob/main/packages/story-editor/src/components/panels/design/videoOptions/videoOptions.js). In fact is mostly a repeat of that code. Why not make this code more reusable and use it in both places? It will make future changes and refactors much easier.

Reply:
That's exactly what I had done and wanted you to review. And forgot to push 🤦 .

So, here it is (for real this time). Sorry for the inconvenience!

Reply:
Any thoughts on using a less generic name? At first glance `useVideoTranscoding` could easily be mistaken for `useFFmpeg` (in fact, that is its former name).

Maybe even `useVideoElementTranscoding` is an improvement, but I'm sure there are better names.